ansible/inventory

```diff
@@ -0,0 +1,3 @@
+[clients]
```

Comment:
please add `ansible/collections/requirements.yml` since you are using `community.general.alternatives` below